### PR TITLE
Add Second OSS Demo announcement with YouTube link

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add Second OSS Demo announcement with YouTube link

### CODE REVIEW

File: news/_posts/2020-11-27-2020-call-for-papers-quick-tour.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ date:  2020-11-27 16:00:00
 authors: [AntoineThomas]
 icon: icon-video-camera
 image: /assets/images/theme/meta-logo-build.png
-tags: [contribute, events, community]
+tags: [contribute, event, community]
```
Comment (matks) [inline news/_posts/2020-11-27-2020-call-for-papers-quick-tour.md:9]: Fixing the tag

File: news/_posts/2021-02-19-first-public-oss-demo.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ subtitle: "We held the first session of the open source project's public demo"
 date:   2021-02-19 16:00:00
 authors: [ MathieuFerment, PabloBorowicz ]
 icon: icon-video-camera
-tags: [contribute, events, community]
+tags: [contribute, event, community]
```
Comment (matks) [inline news/_posts/2021-02-19-first-public-oss-demo.md:8]: Fixing the tag

**Review by ttoine — APPROVED** [review]

File: news/_posts/2021-02-24-meet-us-second-oss-demo.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Second public demo from the maintainers team"
+subtitle: "Meet us at the second session of the open source project's public demo"
```
Comment (eternoendless) [inline news/_posts/2021-02-24-meet-us-second-oss-demo.md]:
```suggestion
subtitle: "Meet us for the second session of the open source project's public demo"
```

File: news/_posts/2021-02-24-meet-us-second-oss-demo.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+date:   2021-02-24 13:00:00
+authors: [ MathieuFerment, PabloBorowicz ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+As announced [previously](https://build.prestashop.com/news/first-public-oss-demo/), the next session of public demonstration from the maintainers team will happen at 2pm CET on tomorrow, Thursday 25th of February!
```
Comment (eternoendless) [inline news/_posts/2021-02-24-meet-us-second-oss-demo.md]:
```suggestion
As announced [previously](https://build.prestashop.com/news/first-public-oss-demo/), the next session of public demonstration from the maintainers team will happen tomorrow, Thursday 25th of February, at 2 pm CET!
```

File: news/_posts/2021-02-24-meet-us-second-oss-demo.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Second public demo from the maintainers team"
+subtitle: "Meet us at the second session of the open source project's public demo"
+date:   2021-02-24 13:00:00
+authors: [ MathieuFerment, PabloBorowicz ]
```
Comment (eternoendless) [inline news/_posts/2021-02-24-meet-us-second-oss-demo.md]:
```suggestion
authors: [ MathieuFerment ]
```

This one you wrote all by yourself. Alternatively, you can also sign it as the prestashop team instead

File: news/_posts/2021-02-24-meet-us-second-oss-demo.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+As announced [previously](https://build.prestashop.com/news/first-public-oss-demo/), the next session of public demonstration from the maintainers team will happen at 2pm CET on tomorrow, Thursday 25th of February!
+
+It will be streamed live on our newly created YouTube channel. Please find the link to join us below:
+
+[Join us on YouTube!](https://youtu.be/JBvJgoo-rLg){: .btn .btn-lg .btn-info}
+{: .text-center}
+
+The session will be kept in record on YouTube, so that even if you can't attend to the live stream, you will still be able to catch up later.
+
+We truly hope to meet you tomorrow.
```
Comment (eternoendless) [inline news/_posts/2021-02-24-meet-us-second-oss-demo.md]:
```suggestion
See you tomorrow!
```

Sounds desperate otherwise 😛

**Comment by matks** [comment]
Thank you for review !

**Pull request merged**